### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +89,24 @@ BinDeps:
...
+
+## Using an already existing Conda installation
+To use an already existing Conda installation, first create an environment for
+`Conda.jl` and then set `CONDA_JL_HOME` to the full path of the environment.
+You have to build `Conda.jl` and all the packages using `Conda.jl` after this.
+
+```shell
+conda create -n conda_jl python
+export CONDA_JL_HOME="/path/to/miniconda/envs/conda_jl"
+julia -e "Pkg.build('Conda')"
```
Comment (tkelman) [inline README.md]: quotes wrong way around, julia strings for Pkg.build need double quotes

File: README.md [hunk 2]
```diff
@@ -30,13 +30,28 @@ You can install this package by running `Pkg.add("Conda")` at the Julia prompt.
...
+- `Conda.list(env)`: list all installed packages.
+- `Conda.add_channel(channel, env)`: add a channel to the list of channels;
+- `Conda.channels(env)`: get the current list of channels;
+- `Conda.rm_channel(channel, env)`: remove a channel from the list of channels;
+
+Parameter `env` is optional and defaults to `ROOTENV`. See below for more info.
+
+### Conda Environments
+
+You can use Conda environments via `Conda.jl`. Note that python packages should
```
Comment (stevengj) [inline README.md]: A few more words of explanation would be helpful, e.g.
```
[Conda environments](http://conda.pydata.org/docs/using/envs.html) allow you to manage multiple
distinct sets of packages in a way that avoids conflicts and allows you to install different versions
of packages simultaneously.  The `Conda.jl` package supports environments by allowing you
to pass an optional `env` parameter to functions for package installation, update, and so on.
If this parameter is not specified, then the default "root" environment (corresponding to the path in `Conda.ROOTENV`) is used.  (If you are installing Python packages for use with [PyCall](https://github.com/JuliaPy/PyCall.jl), you must use the root environment.)   The
environment name can be specified as a `Symbol`, or the full path of the environment
(if you want to use an environment in a nonstandard directory) can be passed as a string.

For example:
```

File: README.md [hunk 3]
```diff
@@ -74,6 +89,24 @@ BinDeps:
 provides(Conda.Manager, "libnetcdf", netcdf, os=:Linux)
 ```
 
+To tell BinDeps to install the package to an environments, use `EnvManager`.
```
Comment (stevengj) [inline README.md]: an environment different from the root environment

File: README.md [hunk 4]
```diff
@@ -74,6 +89,24 @@ BinDeps:
...
+
+```julia
+provides(Conda.EnvManager{:my_env}, "libnetcdf", netcdf)
+```
+
+## Using an already existing Conda installation
+To use an already existing Conda installation, first create an environment for
+`Conda.jl` and then set `CONDA_JL_HOME` to the full path of the environment.
+You have to build `Conda.jl` and all the packages using `Conda.jl` after this.
```
Comment (stevengj) [inline README.md]: set the `CONDA_JL_HOME` environment variable

build -> rebuild

**Pull request merged**